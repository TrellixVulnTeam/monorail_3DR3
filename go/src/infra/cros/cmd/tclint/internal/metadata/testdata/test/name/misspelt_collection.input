{
    "remoteTestDrivers": [
        {
            "name": "remoteTestDrivers/tauto",
            "tests": [
                {
                    "name": "remoteTestDrivers/tauto/test/foo"
                }
            ]
        }
    ]
}