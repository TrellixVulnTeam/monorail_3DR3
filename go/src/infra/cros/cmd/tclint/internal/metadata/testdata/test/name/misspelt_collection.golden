[
"## Errors ##",
"Test 'remoteTestDrivers/tauto/test/foo': name must be of the form 'tests/{test}', missing prefix 'tests'"
]